connector/connectionFields.xml

```diff
@@ -11,6 +19,29 @@
 
   <field name="password" label="Password" value-type="string" category="authentication" secure="true" />
 
+  <field name="v-authDatabase" label="Authentication Database" value-type="string" category="authentication" default-value="admin" />
```

Comment:
I share some of the reserve Ryan has here. It would be nice to be able to let the user also input his custom jdbc url, because otherwise we are restricting them to only the parameters that are in the dialog. These here are the ones we use, but other users can have more refined urls best suited for their needs. 
The issue I see is that not all the parameters are supported in the jdbc url. dialect isn't (but it's not visible for Tableau users, so not a blocker), logging params won't be though unless we add support for them in the jdbc driver itself. 
In short, if not all parameters are supported in the url, we can't rely on only letting the user provide its url. Maybe a conditional display which only "asks" for the parameters which are not supported in the url when opting for the non default url would do the trick? 
I'm also (still) very much in-favour of supporting all parameters as part of the jdbc url ;)

Reply:
Spoke with Ryan offline and as the current iteration has few properties to set such as 'database', I will make the change to allow the user to provide the mongodb URI and for the additional properties we can discuss the design of when we get to it.